### PR TITLE
Adopt fix for 1.16 and tls mssqldb panic

### PR DESCRIPTION
**Pull request by rmfitzpatrick**
Resolution to https://github.com/denisenkom/go-mssqldb/issues/639.

Thanks to @dloucasfx for finding.

### CODE REVIEW

**Comment by rmfitzpatrick** [comment]
Validated in Splunk Collector distribution via https://github.com/signalfx/splunk-otel-collector/pull/481

**Review by mstumpfx — APPROVED** [review]

File: go.sum [hunk 1]
```diff
@@ -361,6 +361,8 @@ github.com/davecgh/go-xdr v0.0.0-20161123171359-e6a2ba005892/go.mod h1:CTDl0pzVz
 github.com/denis-tingajkin/go-header v0.4.2/go.mod h1:eLRHAVXzE5atsKAnNRDB90WHCFFnBUn4RN0nRcs1LJA=
 github.com/denisenkom/go-mssqldb v0.0.0-20200428022330-06a60b6afbbc h1:VRRKCwnzqk8QCaRC4os14xoKDdbHqqlJtJA0oc1ZAjg=
```
Comment (pmcollins) [inline go.sum:362]: Looks like the previous version's entries are still here

**Review by pmcollins — APPROVED** [review]
LGTM, just one observation/question (outside of my golang knowledge)

File: go.sum (hunk 1 above)
Comment (rmfitzpatrick) [reply go.sum]: Tidied to remove, thanks

**Pull request merged**